data/profiles/default.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: install.istio.io/v1alpha2
 kind: IstioControlPlane
 spec:
   hub: gcr.io/istio-release
-  tag: master-latest-daily
+  tag: release-1.3-latest-daily
```

Comment:
need to ensure this tag is available at some point or the operator will be doa in the firs trc.

Reply:
Good point.
@howardjohn do you know what needs to be done to get those daily builds created?